### ISSUE TITLE
github org for tests?

### ISSUE THREAD

**Issue by dbalsom** [TomHarte/ProcessorTests#83]
This repo is getting a bit huge; and I will have another set to contribute soon which is going to make it even bigger.

What would you think about making a GitHub organization for the CPU tests and having each architecture in its own repo?  That would reduce the requirement for people to check out multiple gigabytes, and modifications/updates to one set/repo wouldn't bloat all of them.

**Comment by TomHarte** [comment]
I think this is definitely the right thing to do; as well as separating the unrelated pieces of work for heft and for tracking, it'll resolve the implicit misattribution of *TomHarte*/ProcessorTests which I've become increasingly uncomfortable about, and provide somewhere just to fork other repositories to where other authors want to contribute similar test sets but to retain full ownership. Also it'll open the door to similar test sets for things that aren't processors.

So I guess the smartest process is:
1. create organisation;
2. transfer this repository to that organisation;
3. manually copy per-processor contents into their own repositories (assumption here: the git history isn't sufficiently interesting to justify the weight of its preservation);
4. add suitable redirects and mark this one as archived.

Does that sound reasonable?

**Comment by dbalsom** [comment]
Sounds good to me.

**Comment by TomHarte** [comment]
Steps (1) and (2) are complete, with the hope that I don't think up a better organisation name any time soon; (3) and (4) remain TODO and may not happen until the weekend or later. But the plan hasn't changed.

**Comment by dbalsom** [comment]
If you'd like, I can reupload the 8088 tests into a new repo under this org.
Is the naming convention to use [CpuModel]Tests? 

should we add raddad772 (jsmoo) to the org as well?

**Comment by TomHarte** [comment]
I don't think we need the 'Tests' suffix; the existing Gameboy repository has just kept its original name, being merely a fork. `SingleStepTests/8088` sounds appropriate to me, but obviously if you have a better idea then run with it.
